src/shared/filesystemUtilities.ts

```diff
@@ -156,6 +156,35 @@ export function isInDirectory(d: string, p: string): boolean {
     })
 }
 
+/**
+ *
+ * @returns file distance between fileA and fileB
+ * For example:
+ * The file distance between A/B/C.java and A/B/D.java is 0
+ * The file distance between A/B/C.java and A/D.java is 1
+ */
+export function getFileDistance(fileA: string, fileB: string): number {
+    let filePathA = pathutils.normalize(fileA).split('/')
```

Comment:
Does this work in Windows?  Isn't it `C:\User\FILE `in windows?

Reply:
To be honest, you should compute the distance from the tree node. Get their lowest common parent node then distance = parent node -> node A + parent node -> node B

Reply:
If you only compare the name of the folder, you are assuming the folder that has same name is the same folder. 

Consider this case:

> /usr/a.py
> /usr/usr/usr/usr/usr/b.py 

Their common parent node is /usr. 
Distance is 4.

Reply:
@leigaol yea we have windows test case https://github.com/aws/aws-toolkit-vscode/pull/3638/files#diff-2900a441114e12a005aa88ef773329ae5c12286bde6cb43b7cc4ce9de441c6a0R177, `normalize` will take care of it. Also I added the test case you mentioned, it should work
```
/usr/a.py
/usr/usr/usr/usr/usr/b.py
```